npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -203,14 +205,23 @@ export async function getAngularJson (projectRoot: string): Promise<AngularJson>
   return JSON.parse(angularJson)
 }
 
+function buildLogger (name: string = ''): logging.Logger {
+  const debugLogger = debugLib(`${debugPrefix}${name ? `:${name}` : ''}`)
+
+  return {
```

Comment:
Cannot find any docs in Angular about the logger, nice job figuring this out.

Another option if we want to cover all bases might be using a `Proxy` and pointing everything that's not one of the properties you defined here as a no-op. Then if someone does something exotic like `logger.subscribe()` it wouldn't error (it also wouldn't do what they are expecting, though).

Will let you make the final call, just throwing another suggestion out there.

Reply:
Took another look based on your suggestion and I ended up finding a better alternative - just build an actual `Logger` instance (which is really just a tracker for `LogEntry` items) and build a subscription that forwards everything to the debug logger. No idea why we weren't just building a `Logger` instance to begin with, but now we're fully compliant with the contract *and* get inspectability from the debug logger